news/2020-07-16-mwc-4k-2020-registrations-open.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: "osu!mania 4K World Cup 2020: Registrations now open!"
+date: 2020-07-16 20:00:00 +0000
+---
+
+Oh boy, here we go again. The osu!mania 4K World Cup registration phase is now open!
+
+![](/wiki/shared/news/banners/MWC4k2020.jpg)
+
+For more details, please refer to the **[osu!mania 4K World Cup 2020 wiki article](/wiki/Tournaments/MWC/2020_4K)**.
+You can discuss this event and follow the most important changes in the **[official discussion thread](https://osu.ppy.sh/community/forums/topics/1106843)** or by joining us at the public **[community Discord server](https://discord.gg/0Vxo9AsejDkGlk3H)**.
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | Jul 16th – Jul 30th |
+| Drawings | Aug 8th (14:00 UTC) |
+| Group Stage | Aug 15th & Aug 16th |
+| Round of 16 | Aug 22nd & May 23rd |
+| Quarterfinals | Aug 29th & Aug 30th |
+| Semifinals | Sep 5th & Sep 6th |
+| Finals | Sep 12th & Sep 13th |
+| Grand Finals | Sep 19th & Sep 20th |
```

Comment:
might wanna consider using same date format as on the wiki?

Reply:
Is just for simplicity sake on news article